README.md

```diff
@@ -55,13 +55,13 @@ Compliance Masonry is a command-line interface (CLI) that allows users to constr
       - ./standard-1.yaml
     dependencies:
       certifications: # An optional list of certifications stored remotely
-        - url: github.com/18F/LATO
+        - url: https://github.com/18F/LATO
```

Comment:
https://github.com/18F/LATO is not a publicly accessible URL. Does LATO live somewhere we can access it?